drivers/manual_hmis_data/app/views/manual_hmis_data/funders/edit.haml

```diff
@@ -1,5 +1,5 @@
 = content_for :modal_title do
-  Edit Funder for #{@funder.project.name} at #{@funder.project.organization.name}
+  Edit Funder for #{@funder.project.name(current_user)} at #{@funder.project.organization.name}
```

Comment:
I _think_ you need `can_edit_projects?` to get to this page.  Perhaps we should pass that in as well?

Reply:
Ah yes, good catch. I'll do what I did for the similar `app/views/inventories/edit.haml`, which is just to call `name(ignore_confidential_status: true)` because if you're able to edit the project, you should be able to see the name.

Reply:
https://github.com/greenriver/hmis-warehouse/pull/2225/commits/af281721df0690997516f102a6aace63879f6617